lesson_2_Section_30_new.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<p>I'm envisioning 4 proposed dots here for Jan 1 2022 right?  I think this is perfect</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
4 is what I had in mind yep